spyder/plugins/completion/kite/plugin.py

```diff
@@ -113,6 +113,12 @@ def _locate_kite_darwin():
     @staticmethod
     def _is_proc_kite(proc):
```

Comment:
@goanpeca, please remove the changes you made to this method in this PR because they are already part of PR #10070.